CHANGES.rst

```diff
@@ -29,29 +29,36 @@ CHANGES
 
     Example: if your test file looked like this ::
 
-        # my/package/tests/test_all.py
-        from zope.app.testing.functional import defineLayer
-        from zope.app.testing.functional import FunctionalDocFileSuite
-        defineLayer('MyFtestLayer', 'ftesting.zcml', allow_teardown=True)
+        # my/package/tests.py
+        from unittest import TestSuite
+        import doctest
+        import zope.app.wsgi.testlayer
+
+        layer = zope.app.wsgi.testlayer.BrowserLayer(my.package)
 
         def test_suite():
-            suite = FunctionalDocFileSuite('test.txt', ...)
-            suite.layer = MyFtestLayer
-            return suite
+            suite = doctest.DocFileSuite('test.txt', ...)
+            suite.layer = layer
+            return TestSuite((suite,))
 
```

Comment:
Oh ... sorry. 🍅 Reverted that part. I'm just too young and didn't know this style. And I thought, when I'm fixing the CHANGELOG I want to bring it into a format that 100% works (old / new are a shameless plug of the `zope.globalrequest` migration).